libs/gi/art-scanner/src/lib/parse.ts

```diff
@@ -108,7 +108,7 @@ export function parseSubstats(texts: string[]): ISubstat[] {
         matches.push({
           key,
           value: parseFloat(
-            match[1].replace(/,/g, '.').replace(/\.{2,}/g, '.')
```

Comment:
Some European countries use a comma in decimal, like `HP+4,1%`. This code intended to to convert that to `HP+4.1%`.
However, you are right in that the issue exists here, `HP+1,016` -> `HP+1.016`.
Since flat stats are rounded, I think the solution is to only apply the `,`->`''` logic on flat stats (or only on flat HP)